### PR TITLE
#4258: Remove quickbar after invalidation

### PR DESCRIPTION
**Pull request by fregante**
## What does this PR do?

- Part of https://github.com/pixiebrix/pixiebrix-extension/issues/4258
- Related to https://github.com/pixiebrix/pixiebrix-extension/pull/6851

This solves a few issues that arise from stale quickbar instances. 

## Before

Notice how further shortcuts don't necessarily toggle the quickbar off, it takes several attempts.

https://github.com/pixiebrix/pixiebrix-extension/assets/1402241/c1e00652-81a8-4f9e-8fc3-bac0cf4c8e73

## After

The quickbar immediately disappears on invalidation, and the new version immediately works correctly.

https://github.com/pixiebrix/pixiebrix-extension/assets/1402241/115b802d-4fac-4c1c-8be9-ed95590e074d

## Checklist

- [x] Designate a primary reviewer: @BLoe

### CODE REVIEW

File: src/components/quickBar/QuickBarApp.tsx [hunk 1]
```diff
@@ -30,7 +30,10 @@ import EmotionShadowRoot from "react-shadow/emotion";
 import faStyleSheet from "@fortawesome/fontawesome-svg-core/styles.css?loadAsUrl";
```
Comment (fregante) [inline src/components/quickBar/QuickBarApp.tsx:30]: Review tip: "Hide whitespace changes"

File: src/components/quickBar/QuickBarApp.tsx [hunk 2]
```diff
@@ -198,11 +209,16 @@ export const initQuickBarApp = once(async () => {
   }
 
   const container = document.createElement("div");
-  container.id = PIXIEBRIX_QUICK_BAR_CONTAINER_ID;
+  container.className = PIXIEBRIX_QUICK_BAR_CONTAINER_CLASS;
```
Comment (fregante) [inline src/components/quickBar/QuickBarApp.tsx:212]: Additional change:

- always prefer classes to IDs; we had multiple instances of this element

File: src/utils/notify.tsx [hunk 3]
```diff
@@ -124,6 +127,7 @@ function getMessageDisplayTimeMs(message: string): number {
 
 export function initToaster(): void {
   const root = document.createElement("div");
+  root.className = PIXIEBRIX_NOTIFICATION_CLASS; // Used to visually identify the element in the DOM
```
Comment (fregante) [inline src/utils/notify.tsx:130]: Additional change:

- add a class to the notification wrapper too

**Review by fregante — COMMENTED** [review]
I added a couple of classes to make the DOM clearer when debugging as well:

### Before

Screenshot 21


### After

Screenshot 20

**Review by BLoe — APPROVED** [review]

**Pull request merged**